apps/web/lib/i18n/locales/en/translation.json

```diff
@@ -101,9 +101,16 @@
       "interface_lang": "Interface Language",
       "user_settings": {
         "user_settings_updated": "User settings have been updated!",
-        "boomark_click_action": "Bookmark Click Action",
-        "open_external_url": "Open Original URL",
-        "open_bookmark_details": "Open Bookmark Details"
+        "bookmark_click_action": {
+          "title": "Bookmark Click Action",
+          "open_external_url": "Open Original URL",
+          "open_bookmark_details": "Open Bookmark Details"
+        },
+        "archive_display_behaviour": {
+          "title": "Archive Display Mode",
+          "show": "Show up in tags and lists",
+          "hide": "Hide in tags and lists"
+        }
```

Comment:
thought it might be a little nicer to nest related text for each setting instead of having them all flat. but im totally fine to revert this change as well